packages/catalog-client/src/types.ts

```diff
@@ -53,3 +53,17 @@ export type AddLocationResponse = {
 export type DiscoveryApi = {
   getBaseUrl(pluginId: string): Promise<string>;
 };
+/**
+ * This is a copy of the core IdentityApi, to avoid importing core.
+ */
+export type ProfileInfo = {
+  email?: string;
+  displayName?: string;
+  picture?: string;
+};
+export type IdentityApi = {
```

Comment:
No need to replicate the full interface here imo, could just include to the `getIdToken` method and skip the rest.

Although if we move over to having a wrapping client in frontend code we won't need this extra type